codegen/projections/high_score_service/lib/high_score_service/client.rb

```diff
@@ -63,10 +63,11 @@ def api_key_auth(params = {}, options = {})
         error_inspector_class: Hearth::HTTP::ErrorInspector
       )
       stack.use(Hearth::Middleware::Auth,
-        auth_schemes: options.fetch(:auth_schemes, config.auth_schemes),
         auth_params: Auth::Params.new(operation_name: :api_key_auth),
         http_api_key_identity_resolver: options.fetch(:http_api_key_identity_resolver, config.http_api_key_identity_resolver),
         auth_resolver: options.fetch(:auth_resolver, config.auth_resolver),
+        identity_resolver_map: { http_login_identity_resolver: Hearth::Identities::HTTPLogin, http_bearer_identity_resolver: Hearth::Identities::HTTPBearer, http_api_key_identity_resolver: Hearth::Identities::HTTPApiKey },
+        auth_schemes: options.fetch(:auth_schemes, config.auth_schemes),
```

Comment:
should we be concating the auth_schemes rather than entirely replacing them?  (I think replacing them is probably fine, but worth double checking)

Reply:
I think it's a replacement but I can check.

Reply:
Yeah, I think it's a replacement. It's supposed to be a map but I chose a list for simplicity. I think this forces an operation to only use certain auth schemes.